app/services/analytics_events.rb

```diff
@@ -534,6 +535,7 @@ def email_and_password_auth(
     rate_limited:,
     valid_captcha_result:,
     captcha_validation_performed:,
+    device:,
```

Comment:
Is this is an instance of the `Device` model? I'm not sure how well that will log.

Reply:
Honestly, I was throwing ideas at the wall. I did not consider the model as my first idea.